apps/calendar/js/views/settings.js

```diff
@@ -144,71 +149,38 @@ Settings.prototype = {
   },
 
   _observeAccountStore: function() {
-    var store = core.storeFactory.get('Account');
-    var handler = this._updateSyncButton.bind(this);
-
-    store.on('add', handler);
-    store.on('remove', handler);
+    var stream = core.bridge.observeSyncableAccounts();
```

Comment:
now that I'm thinking about it, this should actually listen for all the accounts. only place where `syncableAccounts` is actually needed is to toggle the sync button display.